web/modules/custom/joinup_core/joinup_core.module

```diff
@@ -274,6 +278,19 @@ function joinup_core_entity_base_field_info(EntityTypeInterface $entity_type) {
       ->setComputed(TRUE);
   }
 
+  // Add the 'subscription_bundles' property to the 'og_membership' entity.
+  if (($entity_type->id() === 'og_membership')) {
+    $fields['subscription_bundles'] = BaseFieldDefinition::create('map')
```

Comment:
As written in the ticket, this will create a serialised field with no easy way of filtering by query.